modules/sbo-advanced-binding-options.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-advanced-binding-options_{context}"]
+= Advanced binding options
+
+This section describes the advanced binding options, which are available only for the `binding.operators.coreos.com` API group.
```

Comment:
```suggestion
Advanced binding options are available only for the `binding.operators.coreos.com` API group.
```
I'm not sure you really need the "this section describes" phrasing, which doesn't really add any meaningful content. Your call on whether you think it's necessary.